dtreeviz/models/spark_decision_tree.py

```diff
@@ -6,8 +6,13 @@
 import pyspark
 
 from dtreeviz.models.shadow_decision_tree import ShadowDecTree
-from pyspark.ml.classification import DecisionTreeClassificationModel
-from pyspark.ml.regression import DecisionTreeRegressionModel
+try:
```

Comment:
This file is specifically about spark so why can't we just do the import? why do we need try/catch.

Reply:
I think this is correct.

Reply:
this was actually my suggestion. It serves more like a  warning message for the user... if he installed dtreeviz without pyspark and latter needed this code, to have this message... 

if I think twice, it doesn't add too much value.... we can refactor it if the code became too verbose using it.

Reply:
ah. I see.  That's a good thought, but I think if Python tells them there is no pyspark, they will get the message.  Might be better to simplify the code.